charts/influxdb2/values.yaml

```diff
@@ -0,0 +1,55 @@
+image:
+  repository: quay.io/influxdb/influxdb
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
```

Comment:
shouldn't this be 9999?

Reply:
The service will be available on port 80, it redirects to the pods 9999 port